src/Plan/Plan.php

```diff
@@ -6,10 +6,14 @@
 
 use Laravel\Cashier\Order\OrderItemPreprocessorCollection;
 use Laravel\Cashier\Plan\Contracts\Plan as PlanContract;
+use Laravel\Cashier\Traits\PaymentMethodString;
```

Comment:
This trait is specific to the FirstPayment, so please move it to the Laravel\Cashier\FirstPayment namespace.